jjb/jobs/qcs-container-install.yaml

```diff
@@ -0,0 +1,82 @@
+- job-template:
+    name: 'qcs-container-install-{distro}'
+    node: '{distro}-os'
+    triggers:
+        - timed: '@midnight'
+    scm:
+        - git:
+            url: https://github.com/quipucords/ci.git
+            basedir: ci
+            skip-tag: true
+            branches:
+              - origin/issues/50
+    wrappers:
+        - inject:
+            properties-content: |
+                DISTRO={distro}
+        - credentials-binding:
+            - file:
+                credential-id: 4c692211-c5e1-4354-8e1b-b9d0276c29d9
+                variable: ID_JENKINS_RSA
+        - ssh-agent-credentials:
+            users:
+                - '390bdc1f-73c6-457e-81de-9e794478e0e'
+    builders:
+        - config-file-provider:
+            files:
+              - file-id: '62cf0ccc-220e-4177-9eab-f39701bff8d7'
+                target: 'camayoc/config.yaml'
+
+        - shining-panda:
+            build-environment: virtualenv
+            python-version: Default-System-Python2
```

Comment:
I added this because RHEL6 has python `2.6` and RHEL7 has `2.7`, so I made this one use whatever `/usr/bin/python2` is.